modules/yieldbotBidAdapter.js

```diff
@@ -238,7 +238,9 @@ export const YieldbotAdapter = {
    */
   urlPrefix: function(prefix) {
     const cookieName = this.CONSTANTS.COOKIES.URL_PREFIX;
-    let cookieValue = prefix || this.getCookie(cookieName);
+    const pIdx = prefix ? prefix.indexOf(':') : -1;
+    const url = pIdx !== -1 ? document.location.protocol + prefix.substr(pIdx + 1) : null;
```

Comment:
This function would have been the instigator of mixed content errors. The server response will include `http:` protocol so on secure sites this would be an issue.

For now we'll add the protocol from the document location (_as with the previous adapter code path_); however, will look further into using a server response `url_prefix` w/o protocol specified. That would enable us to eliminate some logic in this function.

Reply:
`/me` note to self... and probable good recommendation for contribution guidelines to run the karma test suite over both protocols:  `http(s)`.

[config.protocol](https://github.com/karma-runner/karma/blob/master/docs/config/01-configuration-file.md#protocol)
[config.httpsServerOptions](https://github.com/karma-runner/karma/blob/master/docs/config/01-configuration-file.md#httpsserveroptions)